2 - RTB/Documentazione esterna/Verbali/23-12-06/23-12-06.typ

```diff
@@ -0,0 +1,60 @@
+#import "/template.typ": *
+
+//change title
+#show: project.with(
+  date: "06/12/23",
+  subTitle: "",
+  docType: "verbale",
+  externalParticipants : (
+    (name: "Matteo Bassani", role: "Referente aziendale"),
+  ), // Rimuovere se non ci sono partecipanti esterni
+  authors: (
+    "Todesco Mattia",
+  ),
+  reviewers: (
+    "Nardo Silvio",
+  ),
+  timeStart: "16:00",
+  timeEnd: "16:40",
+);
+
+= Ordine del giorno
+- discussione sulle tecnologie implementate e da implementare;
+- dimostrazioni tecniche tramite screen sharing dei PoC A e B;
+- chiarimenti tecnici sull'implementazione dei PoC;
+- discussione eventuale prossimo meeting.
+
+== Tecnologie Implementate
+Sono state presentate al Proponente le tecnologie attualmente implementate, e precedentemente discusse con lo stesso, e sono stati dati riferimenti per quanto riguarda le tecnologie ancora da implementare:
+- fatte: Three.js, MySQL;
+- da fare: Docker, Next.js.
+
+== Dimostrazioni Tecniche dei PoC A e B
+Sono stati presentati al Proponente due PoC denominati A e B con le seguenti caratteristiche:
+
+- PoC A:
+  - funzionalità implementate:
+        creazione, eliminazione e spostamento di scaffali. UI nativa Three.js con possibilità di definire dimensioni e precisione della "scacchiera";
+
+  - requisiti minimi: zoom in/out e rotazione(rotazione su tre assi opzionale).
+
+Non sono ancora presenti bin.
+
+- PoC B:
+  - funzionalità implementate:
+    - creazione di un magazzino tramite file svg;
```

Comment:
non so se SVG vada scritto in maiuscolo, in caso va cambiato anche più avanti nel file